docs/general/serializers.md

```diff
@@ -74,6 +74,30 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+Polymorphic relationships are serialized just by specifying the relationship. For example:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable
+end
+```
+
+If the instance of `Picture` belongs to a `Product`, an example payload might be:
+
```

Comment:
Yeah, it's really a serializer concern, eg serializer.associations that is presented consistently by adapter